.eslintrc

```diff
@@ -3,5 +3,9 @@
     "extends": "eslint-config-hapi",
     "plugins": [
         "markdown"
-    ]
+    ],
+    "rules": {
+        "strict": 0,
```

Comment:
can't you add `/*eslint strict:0*/` above the tutorial pages? Or is that to much work?